_data/examples.yml

```diff
@@ -82,3 +82,10 @@
   thumbnail: polymake.png
   language: julia
 
+- title: "Example GroebnerBasis.jl"
+  repository: ederc/GroebnerBasisNotebooks
+  filename: GroebnerBasis
+  author: Christian Eder
+  thumbnail: GroebnerBasis.png
```

Comment:
This thumbnail seems to be missing... did you intend to provide it in this pull request?

Reply:
Sorry, you are right. I just added it. Thanks